infrastructure/zk/src/hyperchain_wizard.ts

```diff
@@ -567,6 +575,67 @@ export function getTokens(network: string): L1Token[] {
     }
 }
 
-export const initHyperchainCommand = new Command('init-hyperchain')
-    .description('Initializes a new Hyperchain network')
+async function selectHyperchainConfiguration() {
+    const envs = env.getAvailableEnvsFromFiles();
+
+    const envQuestions = [
+        {
+            message: 'Which Hyperchain configuration do you want to use?',
+            name: 'env',
+            type: 'select',
+            choices: [...envs].sort()
+        }
+    ];
+
+    const envResults: any = await enquirer.prompt(envQuestions);
+    return envResults.env;
+}
+
+async function generateDockerImages() {
+    await warning(`\nThis process will build the docker images and it can take a while. Please be patient.\n`);
+
+    const envName = await selectHyperchainConfiguration();
+
+    await docker.customBuildForHyperchain('server-v2', envName);
+
+    await warning(`\n\n\n\n\nDocker image for server created: Server image: ${envName}/server-v2:latest\n`);
+
+    let hasProver = false;
+
+    if (process.env.ETH_SENDER_SENDER_PROOF_SENDING_MODE !== 'SkipEveryProof') {
+        hasProver = true;
+        // TODO: Hyperchain is using prover, so we must include Boojum images - wait for Boojum merge
```

Comment:
As mentioned above, using real provers requires a different config, so it probably also must be handled.
Also, same w.r.t. `TODO`s and task numbers.